crates/proto/src/rr/rdata/caa.rs

```diff
@@ -242,13 +242,19 @@ fn read_value(
     match *tag {
         Property::Issue | Property::IssueWild => {
             let slice = decoder.read_slice(value_len)?.unverified(/*read_issuer verified as safe*/);
-            let value = read_issuer(slice)?;
-            Ok(Value::Issuer(value.0, value.1))
+            if let Ok(value) = read_issuer(slice) {
```

Comment:
Nit: suggest spelling it more like

```rust
Ok(match read_issuer(slice) {
    Ok(value) => Value::Issuer(value.0, value.1),
    _ => Value::Unknown(slice.to_vec()),
})
```

(Same for the `iodef` case.)

Reply:
Done, and squashed